var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -61,6 +72,8 @@ def headers(self):
 
 
 class AutotoolsBuilder(AutotoolsBuilder):
+    build_directory = "builds/unix"
```

Comment:
Version 2.6.1 passes configure/build but fails during install because it's not in the right directory. I'm hoping this var is correct for all versions.